superset/assets/src/dashboard/components/Dashboard.jsx

```diff
@@ -232,7 +271,14 @@ class Dashboard extends React.PureComponent {
   }
 
   render() {
-    return <DashboardBuilder />;
+    return (
+      <div>
+        <Modal show={this.state.showOmni} ref={this.exampleRef}>
```

Comment:
I think you could create a component with both the modal + `OmniContainer` to simplify this component. and also use a react fragment instead of adding an unneeded `div`

```
<React.Fragment>
  <OmniWithModal />
  <DashboardBuilder />
</React.Fragment>
```

Reply:
What is `React.Fragment` doing for us here?

Reply:
It is a trick to get ride of holder `div`s. 🙂 
See https://reactjs.org/docs/fragments.html